ksp_plugin_adapter/plotter.cs

```diff
@@ -0,0 +1,243 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Runtime.InteropServices;
+
+namespace principia {
+namespace ksp_plugin_adapter {
+
+class Plotter {
+  public Plotter(PrincipiaPluginAdapter adapter) {
+    adapter_ = adapter;
+  }
+
+  public void Plot(DisposablePlanetarium planetarium,
```

Comment:
`PlotCelestialAndVesselTrajectories`, because `Plotter.Plot` is better than `Foo.Bar`, but not much.

Reply:
PlotTrajectories.